tools/deploy-service.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import subprocess
+ 
+def deploy():
+    subprocess.Popen(['sh', 'deploy-script.sh'])
+    
+
+class RequestHandler(BaseHTTPRequestHandler):
+    def handleDeploy(self):
+        # Send response status code
+        self.send_response(200)
+
+        # Send headers
+        self.send_header('Content-type','text/html')
+        self.end_headers()
+
+        message = 'Deploying...'
+
+        # Write content as utf-8 data
+        self.wfile.write(bytes(message, 'utf8'))
+        deploy()
+
+
+    def handleNotFound(self):
+        self.send_response(404)
+        # Send headers
+        self.send_header('Content-type','text/html')
+        self.end_headers()
+
+        message = '404'
+
+        # Write content as utf-8 data
+        self.wfile.write(bytes(message, 'utf8'))
+
+
+    def do_GET(self):
+        if self.path == '/deploy':
```

Comment:
We should add somekind of password to protect our deployment script from being runned by unauthorized people.